vtysh/vtysh.c

```diff
@@ -3396,6 +3396,76 @@ DEFUN (vtysh_show_running_config,
 	return vtysh_write_terminal(self, vty, argc, argv);
 }
 
+static void show_route_map_send(const char *route_map, bool json)
+{
+	unsigned int i;
+	bool first = true;
+	char command_line[128];
```

Comment:
I'm just curious if we don't get a coverity issue here regarding uninitialized array.

Reply:
Probably not, right after this line there is a `snprintf` which fills it with a null terminated string.